js/src/main/scala/scala/json/ast/JValue.scala

```diff
@@ -31,6 +31,34 @@ sealed abstract class JValue extends Product with Serializable {
   def toJsAny: js.Any
 }
 
+@JSExport
+object JValue {
+  /**
+    * Converts a Javascript object/value coming from Javascript to a [[JValue]].
+    *
+    * @return
+    */
+  @JSExport
+  def fromJsAny(json: js.Any): JValue =
```

Comment:
Shouldn't this have a return type of `Option[JValue]`? Not every `js.Any` can be converted to a `JValue`.